cnames_active.js

```diff
@@ -1066,6 +1066,7 @@ var cnames_active = {
   "fortune": "fortunejs.github.io/fortune", // noCF? (don´t add this in a new PR)
   "fosspwa": "foss-pwa.github.io/app-store",
   "fov": "animafps.github.io/fov-utilities",
+  "foxgpt": "coolgoagle.github.io/fox"
```

Comment:
```suggestion
  "foxgpt": "coolgoagle.github.io/fox",
```
As the CI error notes, you are missing a comma at the end of your entry